CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased] (alpha)
 ### Added
 
+- New Flag `--on-error` which controls recoverable error handling behavior.  Accepts `--on-error fail` (default) to fail out immediately, and `--on-error recover` to proceed with the operation.
```

Comment:
The option name of `recover` seems weird. 

My understanding is that this is a more general parameter. A more appropriate value is probably `continue`

Reply:
Good call, will change.